src/chipeitlib/src/main/java/es/chipeit/lib/emulator/Chipeit.kt

```diff
@@ -46,9 +43,16 @@ class Chipeit(
             ShortMemory(ShortArray(16))
     )
 
-    private val cpu = Cpu(memory, graphicMemory, registers, stack)
     private val soundTimer = Timer(soundPlayer)
     private val delayTimer = Timer()
+    private val cpu = Cpu(
```

Comment:
Initialize this members in the same order they are used in the `Chipeit` ctor. (This is my fault anyways).